pkg/Microsoft.ML.AdditionalLearners/Microsoft.ML.AdditionalLearners.nupkgproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET additional learners. They depend on other packages, like MlNetMklDeps. </PackageDescription>
```

Comment:
>additional [](start = 31, length = 10)

The name "additional learners" is something we might want to reconsider, since in future as we publish more wrappers we will surely want descriptive names.

I think naming it `OrdinaryLeastSquares` is probably a fine choice since that's literally all it is?

Reply:
I was thinking that maybe we can have SymSGD here too, and other learners that need to drag dependencies? 

@ivanidzo4ka didn't like it either and suggested MklDepsLearners. 

---
In reply to: [206582236](https://github.com/dotnet/machinelearning/pull/611#discussion_r206582236) [](ancestors = 206582236)

Reply:
renaming it to HalLearners. If that doesn's sound good, please suggest a name :)

---
In reply to: [206634151](https://github.com/dotnet/machinelearning/pull/611#discussion_r206634151) [](ancestors = 206634151,206584367,206582236)

Reply:
I like HAL learners.

---
In reply to: [206661889](https://github.com/dotnet/machinelearning/pull/611#discussion_r206661889) [](ancestors = 206661889,206634151,206584367,206582236)